projections/shapes/lib/shapes/paginators.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+# This is generated code!
+
+module ShapeService
+  # @api private
+  module Paginators
```

Comment:
If the service does not have any paginator traits on their service/operations - i think we should omit these pagination-related files, right?

Reply:
Eh. We could omit them and conditionally add the pagination if we want to. Right now for protocol tests we generate an empty file too. We can do a separate change for both of these and support autoloading after this.